src/Actions/NotifyUsersWorkflowAction.php

```diff
@@ -86,9 +87,21 @@ public function fieldLabels($relations = true)
 
     public function execute(WorkflowInstance $workflow)
     {
+        if (!$this->EmailTemplate) {
+            Injector::inst()->get(LoggerInterface::class)->warning(
+                'No email template available for notify users workflow action '
+                . "'{$this->Title}' in workflow '{$this->WorkflowDef()?->Title}'."
+            );
+            return true;
+        }
```

Comment:
This change fixes https://github.com/silverstripe/.github/issues/392 by logging a warning and continuing.
Note that the precedent of not having a hard error for failures is already set by handling invalid member email addresses by just skipping them.

This change allows content authors to continue (and workflows will continue to operate), while also logging a warning that can be followed up on.